OutOfSchool/OutOfSchool.BusinessLogic/Services/UserService.cs

```diff
@@ -65,7 +65,7 @@ public async Task<ShortUserDto> GetById(string id)
         return mapper.Map<ShortUserDto>(user);
     }
 
-    public async Task<ShortUserDto> Update(ShortUserDto dto)
+    public async Task<BaseUpdateUserDto> Update(BaseUpdateUserDto dto)
```

Comment:
When we separate dtos we need to treat create/update DTOs as contravariant types - they are passed `in` the method, but should not be returned from a method. For that we use presentation (in this case `ShortUserDto`) that can be returned (treat it as a covariant type).